test/http-invocation.test.js

```diff
@@ -198,6 +198,20 @@ describe('HttpInvocation', function() {
       expect(inv.createRequest()).to.eql(expectedReq);
     });
 
+    it('creates a loopback auth request', function() {
+      var inv = givenInvocationForEndpoint(null, [], null,
+        {accessToken: {id: 'abc'}});
+      var expectedReq = { method: 'GET',
+        url: 'http://base/testModel/testMethod',
+        protocol: 'http:',
+        json: true,
+        headers: {
+          Authorization: 'abc'
+        }
+      };
+      expect(inv.createRequest()).to.eql(expectedReq);
```

Comment:
This assert is checking too much. If we change the underlying implementation in the future (e.g. by adding more fields to the returned request), then this test will fail despite the fact that the implementation may still be correct.

Please check only that request headers contain the expected authorization.

```js
expect(inv.createRequest().headers).to.have.property('Authorization', 'abc');
```

Reply:
Np. I was copying all the following methods, but this change is more stable.